samples/Playground.Forms/Playground.Forms/ViewModels/TabbedNavigation/TabbedPageViewModel.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using System.Collections.Generic;
+using System.Windows.Input;
+using Softeq.XToolkit.Common.Commands;
+using Softeq.XToolkit.WhiteLabel.Model;
+using Softeq.XToolkit.WhiteLabel.Mvvm;
+using Softeq.XToolkit.WhiteLabel.Navigation;
+
+namespace Playground.Forms.ViewModels.TabbedNavigation
+{
+    public class TabbedPageViewModel : ViewModelBase, ITabbedViewModel
+    {
+        private readonly IPageNavigationService _pageNavigationService;
+
+        public TabbedPageViewModel(
+            IPageNavigationService pageNavigationService,
+            RootFrameNavigationPageViewModel<Tab1PageViewModel> tab1,
+            RootFrameNavigationPageViewModel<Tab2PageViewModel> tab2)
```

Comment:
Looks like root view model can be resolved via `TabItem<TViewModel>.CreateViewModel()`

Reply:
Improved